Gruntfile.js

```diff
@@ -0,0 +1,206 @@
+module.exports = function (grunt) {
+	'use strict';
+
+	/**
+	* LOAD TASKS
+	*/
+
+	require( 'matchdep' ).filterDev( 'grunt-*' ).forEach( grunt.loadNpmTasks );
+	
+	/**
+	* CONFIGURATION
+	*/
+	grunt.initConfig({
+		pkg: grunt.file.readJSON('package.json'),
+		jshint: {
+			dist: [
+				'js/*.js'
+			],
+			options: {
+				"-W041": false,
+				curly: true,
+				eqeqeq: false,
+				immed: true,
+				latedef: false,
+				newcap: true,
+				noarg: true,
+				sub: true,
+				undef: false,
+				boss: true,
+				eqnull: false,
+				browser: true,
+				jquery: true,
+				globals: {
+					require: true,
+					define: true,
+					requirejs: true,
+					describe: true,
+					expect: true,
+					it: true
+				}
+			}
+		},
+		clean: {
+			options: {
+				force: true
+			},
+			dist: [
+				'css/dist/cloud-search.min.css',
+				'css/dist/cloud-search-suggest.min.css',
+				'css/dist/images/',
+				'js/dist/cloud-search.min.js',
+				'js/dist/cloud-search-admin-help.min.js',
+				'js/dist/cloud-search-admin-manage.min.js',
+				'js/dist/cloud-search-admin-settings.min.js',
+				'js/dist/cloud-search-locale-en-US.min.js',
+				'js/dist/cloud-search-locale-it-IT.min.js',
+				'js/dist/cloud-search-suggest.min.js',
+			],
+			deploy: [
+				'deploy'
+			]
+		},
+		copy: {
+			dist: {
+				files: [
+					{
+						expand: true,
+						cwd: 'css/images/',
+						src: ['**'],
+						dest: 'css/dist/images/'
+					}
+				]
+			},
+			deploy: {
```

Comment:
This does the copy of the plugin files to the 'deploy' directory. Mostly it's just a list of what not to copy (i.e. things like the GitHub README.md file, assets directory (gets submitted separately by the wp_deploy task) and so forth.

Reply:
Ok, it's clear ;-)